cmd/kpod/create.go

```diff
@@ -0,0 +1,899 @@
+package main
+
+import (
+	"fmt"
+
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+	"strings"
+
+	"github.com/docker/go-units"
+	"github.com/kubernetes-incubator/cri-o/libpod"
+	ann "github.com/kubernetes-incubator/cri-o/pkg/annotations"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+	"strconv"
+)
+
+type mountType string
+
+// Type constants
+const (
+	// TypeBind is the type for mounting host dir
+	TypeBind mountType = "bind"
+	// TypeVolume is the type for remote storage volumes
+	TypeVolume mountType = "volume"
+	// TypeTmpfs is the type for mounting tmpfs
+	TypeTmpfs mountType = "tmpfs"
+	// TypeNamedPipe is the type for mounting Windows named pipes
+	TypeNamedPipe mountType = "npipe"
```

Comment:
Do we want to keep this in? It's good for compatibility, but I don't think we ever need to support this, so it might be sufficient to just drop it and let the normal CLI parsing error handling take care of it.

Reply:
Remove it.

Reply:
fixed